narayana-release-process.sh

```diff
@@ -72,8 +72,7 @@ then
   fi
   set -e
   echo "Checking if there were any failed jobs, this may be interactive so please stand by"
-  JIRA_HOST=issues.redhat.com JENKINS_JOBS=narayana,narayana-catelyn,narayana-documentation,narayana-hqstore,narayana-jdbcobjectstore,narayana-quickstarts,narayana-quickstarts-catelyn\
-      ./scripts/release/pre_release.py
+  JIRA_HOST=issues.redhat.com JENKINS_JOBS=EAP73 ./scripts/release/pre_release.py
```

Comment:
I have just renamed this CI job to narayana59 /cc @mmusgrov

Reply:
+1